packages/perseus-editor/src/widgets/expression-editor.tsx

```diff
@@ -9,22 +9,28 @@ import {
 } from "@khanacademy/perseus";
 // eslint-disable-next-line import/no-extraneous-dependencies
 import lens from "hubble";
-import PropTypes from "prop-types";
 import * as React from "react";
 import _ from "underscore";
 
 import SortableArea from "../components/sortable";
 
+import type {PerseusExpressionWidgetOptions} from "@khanacademy/perseus";
+
 const {InfoTip, PropCheckBox, TexButtons} = components;
 const {getDependencies} = Dependencies;
 
-const answerFormType = PropTypes.shape({
-    considered: PropTypes.oneOf(PerseusExpressionAnswerFormConsidered)
-        .isRequired,
-    value: PropTypes.string.isRequired,
-    form: PropTypes.bool.isRequired,
-    simplify: PropTypes.bool.isRequired,
-});
+type Props = {
+    widgetId?: any;
+    value?: string;
+} & Omit<PerseusExpressionWidgetOptions, "buttonsVisible"> &
```

Comment:
Interesting that we don't use this `buttonsVisible` prop.  Looking around, my guess is this is historical and not actually used in our Perseus data... but rather a side effect that bubbled up from the `<MathInput>` that also has this prop.  

Not asking for change, just pondering.